environments/demo/demo-platform-hmcts-net.yml

```diff
@@ -35,6 +35,9 @@ cname:
   - name: www.moneyclaims
     record: demo-waf.platform.hmcts.net.
     ttl: 300
+  - name: cmc-demo-claim-store
+    record: demo-waf.platform.hmcts.net.
```

Comment:
this is not where you want it to go, you want an A record pointing to the ip address that it's pointed to publically

Reply:
done